vnext/src/overrides.json

```diff
@@ -790,14 +790,18 @@
       "type": "patch",
       "file": "RNTester\\js\\components\\RNTesterExampleList.windows.js",
       "baseFile": "RNTester\\js\\components\\RNTesterExampleList.js",
-    "baseVersion": "0.61.5",
+      "baseVersion": "0.61.5",
       "baseHash": "d9b4912cb42b5709362ea8d4e3974641b56e60a8",
       "issue": "LEGACY_FIXME"
     },
     {
       "type": "platform",
       "file": "RNTester\\js\\examples-win\\Accessibility\\AccessibilityExampleWindows.tsx"
     },
+    {
+      "type": "platform",
+      "file": "RNTester\\js\\examples-win\\Accessibility\\AsyncStorageExampleWindows.tsx"
```

Comment:
Wrong path?

Did you edit this manually? CI should hopefully yell at you for this, but you're the first person to add something post-enforcement.

You can manipulate this file using 'yarn override add'